src/main/java/com/synopsys/integration/alert/component/settings/SettingsDescriptorActionApi.java

```diff
@@ -66,7 +66,38 @@ public void validateConfig(final Collection<ConfigField> descriptorFields, final
         super.validateConfig(descriptorFields, fieldModel, fieldErrors);
         validateLDAPSettings(fieldModel, fieldErrors);
     }
-    
+
+    @Override
+    public void testConfig(final Collection<ConfigField> descriptorFields, final TestConfigModel testConfig) throws IntegrationException {
+
+    }
+
+    @Override
+    public FieldModel readConfig(final FieldModel fieldModel) {
+        final Optional<UserModel> defaultUser = userAccessor.getUser("sysadmin");
+        final FieldModel newModel = createFieldModelCopy(fieldModel);
+        final boolean defaultUserPasswordSet = defaultUser.isPresent() && StringUtils.isNotBlank(defaultUser.get().getPassword());
```

Comment:
```java
final boolean defaultUserPasswordSet = defaultUser.map(UserModel::getPassword).filter(StringUtils::isNotBlank).isPresent();
```

Reply:
much cleaner I will change the code in the file.